data_safe_haven/administration/users/active_directory_users.py

```diff
@@ -1,15 +1,17 @@
 """Interact with users in an Azure Active Directory"""
 # Standard library imports
 import pathlib
-from typing import Any, Optional, Sequence
+from collections.abc import Sequence
+from typing import Any
+
+from data_safe_haven.administration.users.research_user import ResearchUser
```

Comment:
This seems to be in the wrong order. If we want to separate same-package imports from same-folder imports, then same-folder ones should come afterwards.

Reply:
I think this happened because of the comment near the other imports and the adding of the absolute path.

If relative imports are allowed the order can be like before

```Python
import pathlib
from collections.abc import Sequence
from typing import Any

from data_safe_haven.config import Config
from data_safe_haven.external import AzureApi
from data_safe_haven.functions import b64encode
from data_safe_haven.pulumi import PulumiSHMStack
from data_safe_haven.utility import FileReader, Logger

from .research_user import ResearchUser
```

Reply:
Fixed in 741290b0